turbo/trie/hashbuilder.go

```diff
@@ -254,6 +273,17 @@ func (hb *HashBuilder) accountLeaf(length int, keyHex []byte, balance *uint256.I
 	if hb.trace {
 		fmt.Printf("Stack depth: %d\n", len(hb.nodeStack))
 	}
+	if hb.accProofResult != nil {
+		// The 'Initialized' and 'Incarnation' fields are presently ignored
+		hb.accProofResult.Balance = (*hexutil.Big)(a.Balance.ToBig())
```

Comment:
- In-theory can read it from PlainState table, instead of capturing it on low-level? Or only here you know what account need to read?

Reply:
Yes, the account information could be obtained at a higher level. It was simpler for me to do it here since I do need the account node from the trie, and it contains all of the necessary data.

Reply:
I have implemented this change, populating the account fields inside eth_call.go rather than in hashbuilder.